aquarius/app/assets.py

```diff
@@ -305,15 +312,38 @@ def validate_remote():
     responses:
       200:
         description: successfully request.
-        example:
-          application/json: true
       400:
         description: Invalid DDO format
       500:
         description: Error
     """
+    if request.content_type != "application/octet-stream":
+        return (
+            jsonify(
+                [
+                    {
+                        "message": "Invalid request content type: should be application/octet-stream"
+                    }
+                ]
+            ),
+            400,
+        )
+
+    pk = os.environ.get("PRIVATE_KEY", None)
+    values = {"hash": "", "publicKey": "", "r": "", "s": "", "v": ""}
+    if pk:
+        raw = request.get_data()
+        wallet = Account.from_key(private_key=pk)
```

Comment:
can you please name this provider_wallet like everywhere else? I also think there is an util callled get_provider_wallet or provider_wallet that would help here